book/src/move-basics/visibility.md

```diff
@@ -92,3 +92,24 @@ fun try_calling_package() {
     package_visibility::package_only();
 }
 ```
+
+## Native Functions
+
+Some functions in the [framework](./../programmability/sui-framework.md) and
+[standard library](./standard-library.md) are marked with the `native` modifier. These functions are
+implemented directly by the Move VM and do not have a body in Move source code. To learn more about
+the native modifier, refer to the
+[Move Reference](/reference/functions.html?highlight=native#native-functions).
+
+```move
+module std::type_name;
+
+public native fun get<T>(): TypeName;
+```
+
+An example from `std::type_name`, learn more about this module in the
```

Comment:
```suggestion
An example from `std::type_name`, learn more about this module in the
```
Not :100: what this references. Maybe start with "This is an..."?

Reply:
Applied, thanks!